detox/package.json

```diff
@@ -24,7 +24,7 @@
     "uuid": "8.3.2"
   },
   "scripts": {
-    "e2e:android-create-emulator": "avdmanager create avd -n detox_emu_api_30 -k 'system-images;android-30;google_apis;x86' -g google_apis -d 'pixel'",
+    "e2e:android-create-emulator": "./start_android_emulator.sh",
```

Comment:
Why start it if I just want to create it?

Reply:
It's [one of the tips in Android local dev from Detox](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#emulator-quick-boot). Though, we're not in full setup with the quick-boot, it still helps save time instead of waiting from emulator's cold-boot, especially during first test. But if we're good with that, I could remove it here for local dev and do it in CI only. If we want it here as well, then maybe should change the name to `e2e:android-start-emulator`. I'm good either way.

Reply:
Okay, I removed the line that starts the emulator. Now, it matches with the current behavior.